model/src/main/java/org/jboss/pnc/model/BuildRecord.java

```diff
@@ -242,6 +242,8 @@
     @Column(name="value")
     private Map<String, String> attributes = new HashMap<>();
 
+    private String repourLog;
```

Comment:
It will create a varchar(256) in database, which is not enough. We should store it as a LOB as in case of build log, so to add annotations:
@Lob
@Type(type = "org.hibernate.type.StringClobType")
@Basic(fetch = FetchType.LAZY)